_component_design/chips.md

```diff
@@ -17,27 +17,63 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## When to use
 
-### When to use
+Use a chips to represent graphically a block of information. For example, use chips to represent a user or device.
```

Comment:
Same note here, I would consider "a user" a unit of information, and device information, such as IP address, port numbers and other related information, a block of information.